### DISCUSSION TITLE
Option to hide the start and/or end time of an event

### DISCUSSION THREAD

**Discussion by ciddi89** [totaldebug/atomic-calendar-revive#1487] [Feature Requests]
Hey,

first at all thanks for the great Integration. I hope my request isn't too special. It would be nice if it is possible to hide the start and/or the end of an event in the event mode. I have for each day several events where the start and end time is exactly the same but I need only the start time of an event.  Or did I miss another way to hide it? 

Thanks 

Christian

<img width="458" alt="Bildschirmfoto 2024-04-26 um 20 31 32" src="https://github.com/totaldebug/atomic-calendar-revive/assets/66088020/c4212ceb-dde3-4153-beff-4ddac7113470">

//Edit: Okay sorry I found #1170. This is exactly the same request.

**Comment by marksie1988** [comment]
As you noted yourself, this is a duplicate of #1170. `showEndTime: false` already hides the end time — see [main configuration docs](https://github.com/totaldebug/atomic-calendar-revive/blob/beta/docs/configuration/main.rst). Closing as already implemented.
